akka-remote/src/main/scala/akka/remote/artery/compress/AdvertiseCompressionId.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery.compress
+
+import akka.actor.Address
+
+/** Callback invoked when a compression id allocation should be advertised to the remote actor system. */
+trait AdvertiseCompressionId[T] {
```

Comment:
could be just a function, but much easier to track it when debugging given a proper name